packages/block-library/src/navigation/edit.js

```diff
@@ -183,6 +176,13 @@ function Navigation( {
 
 export default compose( [
 	withSelect( ( select, { clientId } ) => {
+		const { getBlockSupport } = select( blocksStore );
+		const allowedBlocks = getBlockSupport(
```

Comment:
I think the preference is to use the `getBlockSupport` function from the `blocks` package over the selector (which is used internally by the function).